src/repos.js

```diff
@@ -1,21 +1,22 @@
-export default {
-  'sivakumar-kailasam/github-webook-test-repo': {
-    labels: [
-      'help wanted',
-      'a11y',
-      'bug',
-      'question',
-    ]
-  },
-  'acorncom/ember-hitlist-tester': {
-    labels: [
-      'Help Wanted'
-    ]
-  },
-  'emberjs/ember.js': {
-    labels: [
-      'Needs Help',
-      'Good for New Contributors'
-    ]
-  }
-};
+// NOTE: Ensure the labels are in lower case to make it easier for repo owners
+const standardLabels = [
+  'bug',
+  'help wanted',
+];
+
+let repoList = [
+  'ember-learn/ember-help-wanted',
+  'ember-learn/ember-help-wanted-webhook',
+  'emberjs/guides',
+  'emberjs/website',
+  'emberjs/ember.js',
+  'ember-cli/ember-cli',
+];
+
+let repoHash = {};
+
+repoList.map(repo => {
+  repoHash[repo] = { labels: standardLabels };
```

Comment:
Well projects that don't have standard labels yet can directly define their repo inside the `repoHash`. But unless we want to maintain internal aliasing to group different names given for a label with same intent, we might have to nudge them to adapt to standard labels. I don't see all types of labels coming here anyway.